brainbox/io/one.py

```diff
@@ -359,11 +381,11 @@ def load_passive_rfmap(eid, one=None):
     else:
         one = one or ONE()
         dtypes = {
-            '_iblrig_RFMapStim.raw',
-            '_ibl_passiveRFM.times',
+            '_iblrig_RFMapStim.raw.bin',
+            '_ibl_passiveRFM.times.npy',
         }
 
-        _ = one.load_dataset(eid, dataset_types=dtypes, download_only=True)
+        _ = one.load_datasets(eid, datasets=dtypes, download_only=True)
```

Comment:
This function (`load_passive_rfmap`) should be updated for the new ONE.  Now it doesn't matter whether eid is a UUID or a path.  Instead of calling load_datasets then calling alfio methods it should simply call `one.load_object(eid, 'passiveRFM')` and `one.load_dataset(eid, '_iblrig_RFMapStim.raw.bin')`  This would reduce the number of lines by a lot.

Reply:
Okay, good point, I can refactor this